### PR TITLE
Add metadata prefix to destination internal columns

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Same as #1450 but applied to both raw and normalized tables

## How
Add "_airbyte_" to all columns generated by airbyte

## Pre-merge Checklist
- [x] *Run integration tests*
- [x] *Publish Docker images*

### CODE REVIEW

File: airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java [hunk 1]
```diff
@@ -43,7 +43,8 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultNormalizationRunner.class);
 
-  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:0.1.5";
+  // public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:0.1.6";
```
Comment (ChristopheDuong) [inline airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java:46]: Uncomment before merge

**Comment by ChristopheDuong** [comment]
Do we need to rename the destination table so they don't conflict with the previous ones?

This might require a full-refresh of previously synced tables too...

**Review by cgardens — APPROVED** [review]
nice! thanks for handling moving all this stuff to constants too.

**Review by michel-tricot — COMMENTED** [review]
Great change!

**Pull request merged**